src/args.jl

```diff
@@ -203,7 +203,7 @@ const _series_defaults = KW(
     :markerstrokewidth => 1,
     :markerstrokecolor => :match,
     :markerstrokealpha => nothing,
-    :bins              => :auto,        # number of bins for hists
+    :bins              => :scott,        # number of bins for hists
```

Comment:
Shouldn't `:auto` default to `:scott`, then?

Reply:
Good question - that'd make it a little less transparent what's going on to the user, especially if defaults change, though.